packages/amplify_auth_cognito/ios/Classes/FlutterFetchCognitoSessionResult.swift

```diff
@@ -15,12 +15,13 @@
 
 import Foundation
 import Amplify
+import AmplifyPlugins
 import AWSPluginsCore
 
 struct FlutterFetchCognitoSessionResult {
   var isSignedIn: Bool
-  var userSub: String
-  var identityId: String
+  var userSub: String?
```

Comment:
Instead of returning a nil value can we give an error or exception like iOS stating why the value is not present?

Reply:
This is challenging because the Flutter method channels are only capable of sending back a single response - either a success or an error.  To do what you're suggesting, we would need to send back some sort of generic data structure with data describing the error, and then cast this to an error on the Dart side.  This is something I think we should consider prior to GA, but can't be accomodated as part of this PR.